tests/labhub_test.py

```diff
@@ -117,6 +121,12 @@ def test_is_room_member(self):
         msg.frm.room.occupants = ['batman', 'superman']
         self.assertTrue(LabHub.is_room_member('batman', msg))
 
+    def test_validate_nick(self):
```

Comment:
Since I think the function itself is not needed, so this test function is irrelevant as well..